app/assets/styles/map.scss

```diff
@@ -0,0 +1,29 @@
+#map {
+  height: 100%;
+  width: 100%;
+}
+
+#list {
+  height: 95%;
```

Comment:
honestly, this was a bit of a hack. here's why.

the entire page has height 100%. within the page, there are two main elements which take up vertical space
- the header
- the map/list

if the map/list were height 100%, they would go over the page. so 95% accounts for the fact that the header takes up some room.

i'm totally open to doing this a better way though! any ideas?